src/servers/main.cc

```diff
@@ -648,6 +716,19 @@ Parse(TRTSERVER_ServerOptions* server_options, int argc, char** argv)
       case OPTION_TF_GPU_MEMORY_FRACTION:
         tf_gpu_memory_fraction = ParseFloatOption(optarg);
         break;
+      case OPTION_TF_ADD_VGPU:
+        tf_vgpu = ParseVGPUOption(optarg);
+        if ((tf_vgpu.gpu_device_ >= 0) && (tf_vgpu.num_vgpus_ > 0) &&
```

Comment:
If parsing returns is was successful so don't need to check or return false here.